README.md

```diff
@@ -1,67 +1,66 @@
-﻿# Project Reunion
+﻿# Project Reunion - Calling all Windows desktop apps!

```

Comment:
This "desktop apps" thing is a point in time issue, not a long-term statement. Probably doesn't go in our L1 intro header.